test/langtools/tools/javac/valhalla/lworld-values/TestReflectiveMirrors.java

```diff
@@ -27,9 +27,12 @@
  * @test
  * @bug 8269956
  * @summary  javac should generate `ldc LPoint;` for class literal Point.class
+ * @modules java.base/jdk.internal.value:+open
```

Comment:
This test does not need reflective access.   `@modules java.base/jdk.internal.value` should work.    Same applies to other tests.